runtime/shared/src/main/scala/org/http4s/googleapis/runtime/auth/GoogleOAuth2JwtBearer.scala

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Rename Oauth2 to GoogleOauth2JwtBearer because there are other oauth2 variants; GoogleOAuth2RefreshToken and GoogleOAuth2TokenExchange.